native/src/widget/scrollable.rs

```diff
@@ -510,7 +510,9 @@ pub fn update<Message>(
                 state.scroll(delta, bounds, content_bounds);
 
                 notify_on_scroll(
-                    state,
+                    state.offset_x,
+                    state.offset_y,
+                    &mut state.last_notified,
```

Comment:
Could we pass `&mut State` instead? Harder to pass the wrong arguments that way.

Reply:
Yup :+1: